nw-firestore-schema.ts

```diff
@@ -97,6 +97,13 @@ interface Application {
       ["question number"]: string;
     }; // hackers will have either one or two questions depending on their role
   },
+  score: {
+    WebsiteScore: int,
+    ResumeScore: int,
+    InterestScore: int,
+    PassionScore: int,
```

Comment:
Kind of a nit 😛 should these be lowerCamelCase for consistency?
```suggestion
    websiteScore: int,
    resumeScore: int,
    interestScore: int,
    passionScore: int,
```